<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sound analyzer</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="PlotWidget" name="plotW"/>
    </item>
    <item>
     <widget class="PlotWidget" name="changeW"/>
    </item>
    <item>
     <widget class="QSlider" name="thresholdS">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="singleStep">
       <number>500</number>
      </property>
      <property name="value">
       <number>2000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="infoPTE"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="linkLE"/>
      </item>
      <item>
       <widget class="QPushButton" name="analyzePB">
        <property name="text">
         <string>Analyze</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopPB">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
